### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR fixes a few smaller ui issues.

- [x] Translations (task-name, choose)
- [x] Delete button in the overlay
- [x] Only tim should be sortable
- [x] Icons for running and planned
- [x] security context in ui

Main PR: https://github.com/sulu/sulu/pull/3037

### CODE REVIEW

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js [hunk 1]
```diff
@@ -82,65 +84,82 @@ define([
...
-                            },
-                            matchings: historyFields
+                    contentFilters: {
+                        status: function(content) {
+                            var iconString = 'fa-question';
+                            switch (content) {
+                                case 'planned':
+                                    iconString = 'fa-clock-o';
+                                    break;
+                                case 'started':
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js:144]: Wha are the two new states here?

Comment (wachterjohannes) [reply src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js:144]: @berndhep wants to have own icons for running and planned tasks.

**Comment by danrot** [comment]
I always get a 404 for the following URL: `http://sulu-minimal.lo/bundles/suluautomation/js/css/main.css?v=develop`, because there is a `/js`, which doesn't belong there.

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js [hunk 2]
```diff
@@ -9,15 +9,17 @@
 
 define([
     'underscore',
+    'config',
     'services/suluautomation/task-manager',
     'text!./skeleton.html',
     'text!/admin/api/tasks/fields'
-], function(_, manager, skeletonTemplate, fieldsResponse) {
+], function(_, Config, manager, skeletonTemplate, fieldsResponse) {
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js]: Should be `config`, shouldn't it?

**Review by danrot — CHANGES_REQUESTED** [review]

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
ACK

**Pull request merged**